Source/Core/Core/CoreTiming.cpp

```diff
@@ -322,10 +324,15 @@ void CoreTimingManager::Advance()
     std::pop_heap(m_event_queue.begin(), m_event_queue.end(), std::greater<Event>());
     m_event_queue.pop_back();
 
+    LWDProfiler::GetCurrentProfiler().SetMarker("Throttle");
     Throttle(evt.time);
```

Comment:
Oh... I missed this in the previous PR. 
Calling Throttle per event is kind of useless, as these events are all scheduled to be executed within a few dozen cycles of each other and the later Throttles will do nothing because they are too close. 

Would make more sense to only Throttle once per Advance() call with global timer. That lowers the overhead slightly, because you only have to set the Throttle marker once per `Advance()` call.